lib/resources/example_json/r4_examples_medication_request.rb

```diff
@@ -164,6 +341,3830 @@ module Resources
       ]
     }.freeze
 
+    R4_MEDICATION_REQUEST_PATIENT_BUNDLE ||= {
+      'resourceType': 'Bundle',
+      'id': '40d441d8-d53c-4bd6-b20e-d0c1af3244a3',
+      'type': 'searchset',
+      'link': [
+        {
+          'relation': 'self',
+          'url': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest?patient=12724065'
+        },
+        {
+          'relation': 'next',
+          'url': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest?patient=12724065&-pageContext=T3BlblBsYXRmb3JtRmhpckNvbnRleHQ9dHJ1ZSZwYWdlQ29udGV4dD02MDY4MDUwXzYwNjgwNTFfMTI3MjQwNjVfMTI5MjMwOV8xJmNvbmNlcHQ9Y2hhcnRlZA%3D%3D&-pageDirection=NEXT'
+        }
+      ],
+      'entry': [
+        {
+          'fullUrl': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest/314010287',
```

Comment:
These should be closed endpoints
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest

Reply:
comment addressed